docs/content/tools/acquia-cli.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Acquia CLI Commands"
+aliases:
+  - /en/master/tools/acquia-drush/
+  - /tools/acquia-drush/
+---
+
+
+## Using Acquia Cloud CLI Commands
+
+The [Acquia Cloud CLI](https://github.com/typhonius/acquia_cli) tool allows you 
```

Comment:
The tool is called `Acquia Cli` on it's GitHub page. I suggest we refer to it as such.